example/Vagrantfile

```diff
@@ -0,0 +1,18 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "fredrikhgrelland/hashistack"
+  config.vm.network "private_network", ip: "10.0.3.10"
+
+  # Hashicorp consul ui
+  config.vm.network "forwarded_port", guest: 8500, host: 8500, host_ip: "127.0.0.1"
+
+  # Hashicorp nomad ui
+  config.vm.network "forwarded_port", guest: 4646, host: 4646, host_ip: "127.0.0.1"
+
+  config.vm.provider "virtualbox" do |vb|
+    vb.linked_clone = true
+    vb.memory = 2048
+  end
```

Comment:
I think we need to discuss best which are best options here. @fredrikhgrelland

Reply:
That depends on how much we need. I guess 1024 is minimum for hive and metastore? Lets try to make it as slim as posible

Reply:
Should we set it to 512 considering it's countdash, or just leave it as is?

Reply:
Does that work?

Reply:
Nope. 1024 does.